.github/workflows/lint.jsonnet

```diff
@@ -119,6 +119,10 @@ local pre_commit_ocaml_job =
         // to debug errors in pre-commit, use instead:
         // opam exec -- pre-commit run --verbose --all lint-ocaml || cat /root/.cache/pre-commit/pre-commit.log
         run: |||
+          # When installing ocamlformat.0.26.1 OPAM will try to rebuild some packages
+          # and for that it requires 'autoconf'.
+          apt-get install -y autoconf
+          opam update --yes # so that OPAM knows about ocamlformat.0.26.1
```

Comment:
`opam update` takes awhile, can we bump the ocaml-layer image to latest instead? It installs 0.26.1 by default: https://github.com/returntocorp/ocaml-layer/blob/9171168c6b1d154f14343e86e35cec9357181267/common-config.sh#L71